src/trading/Orders.sol

```diff
@@ -0,0 +1,320 @@
+/**
+ * Copyright (C) 2015 Forecast Foundation OU, full GPL notice in LICENSE
+ */
+
+pragma solidity ^0.4.13;
+
+import 'ROOT/Controller.sol';
+import 'ROOT/libraries/math/SafeMathInt256.sol';
+import 'ROOT/libraries/Trading.sol';
+import 'ROOT/reporting/Interfaces.sol';
+import 'ROOT/trading/OrdersFetcher.sol';
+
+
+/**
+ * @title Orders
+ * @dev Storage of all data associated with orders
+ */
+contract Orders is Controlled {
```

Comment:
This needs to derive from `DelegationTarget` as the first base class: `contract Orders is DelegationTarget, Controlled { ... }`.